README.md

```diff
@@ -54,6 +54,65 @@ demo.rb -- 8 warnings:
   [3]:Dirty#awful has unused parameter 'y' (UnusedParameters)
 ```
 
+## Fixing Smell Warnings
+
+Since `reek` mostly focuses on high-level code smells we can't tell you how to fix warnings in
+a generic fashion since this is and will always be completely dependent on your domain
+language and bussiness logic.
+
+A simple example though might help you get going. Have a look at the sample code below using a
+"Ruby on Rails" model (be aware though that this example is truncated and not really working code):
+
+```Ruby
+class ShoppingCart < ActiveRecord::Base
+  has_many :items
+
+  def gross_price
+    items.sum {|item| item.net + item.tax }
```

Comment:
Let’s make the examples RuboCop-compliant and add a space after the `{` (here and below). :)